server/models/db/db.js

```diff
@@ -0,0 +1,34 @@
+const config = require('../../../config');
```

Comment:
In your config.js you can get the database URL from the .env file for development environment

Reply:
I have updated the files now everything works fine.